.github/workflows/tests.yaml

```diff
@@ -0,0 +1,103 @@
+name: Tests
+on: [push, pull_request]
+
+jobs:
+  packaging:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.6"
+      - run: python -m pip install tox
+      - run: tox -e packaging
+
+  pep8:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.6"
+      - run: python -m pip install tox
+      - run: tox -e pep8
+
+  black:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.6"
+      - run: python -m pip install tox
+      - run: tox -e black
+
+  run-unit-tests-35:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.5"
+      - run: python -m pip install tox
+      - run: tox -e py35
+
+  run-unit-tests-36:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.6"
+      - run: python -m pip install tox
+      - run: tox -e py36
+
+  run-unit-tests-37:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.7"
+      - run: python -m pip install tox
+      - run: tox -e py37
+
+  run-unit-tests-38:
```

Comment:
I think we should also run these tests on 3.9 and the recently released 3.10 (if setup-python supports it)

Reply:
Looks like 3.10 is supported as of about three hours ago! https://github.com/actions/python-versions/releases/tag/3.10.0-117470